src/main/java/com/todoist/markup/EmojiParser.java

```diff
@@ -31,58 +28,37 @@ static Pattern getEmojiPattern() {
     }
 
     static String getEmoji(String key) {
-        return sEmojiMap.get(key);
+        if (sEmojiMap != null && sEmojiMap.has(key)) {
+            return sEmojiMap.getString(key);
+        } else {
+            return null;
+        }
     }
 
+    @SuppressWarnings("unchecked")
     static synchronized void init() throws IOException {
```

Comment:
Another small benefit is that this method doesn't make IO operations anymore so this exception can be removed 😄